app/views/static/subject_example.csv

```diff
@@ -0,0 +1,4 @@
+HEADING,URI,ARTICLE,CATEGORY
+Sally Joseph Carr Brumfield,P001001,Daughter of Rufus Bascom Carr and Fannie Edmonia Hobson,People
```

Comment:
So P001001 breaks the UI a bit, I don't think we have to address it here but I opened issue #1778 to address it later.